explore/src/main/scala/explore/cache/ProgramCache.scala

```diff
@@ -213,6 +241,15 @@ object ProgramCache
               queryProgramTimes
         )
 
+        val groupTimeRangeUpdate: Pipe[
+          IO,
+          ProgramQueriesGQL.GroupEditSubscription.Data,
+          ProgramSummaries => ProgramSummaries
+        ] = keyedSwitchEvalMap(
+          _.groupEdit.value.map(_.id),
+          data => updateGroupTimeRange(props.programId)
```

Comment:
Should this also flatTap to queryProgramTimes?

Reply:
Yes, I've added that now